content/docs/guides/branch-restore.md

```diff
@@ -225,3 +299,5 @@ A couple of details to note:
 
 - You cannot delete a backup branch without first removing the child branch.
 - Once you restore a branch, [Reset from parent](/docs/manage/branches#reset-a-branch-from-parent) restores from the restore backup branch, not the original parent.
+
+  For example, let's say you have a `main` branch with a child development branch `dev/alex`. You are working on `dev/alex` and decide to restore to an earlier point in time to fix something during development. At this point, `dev/alex`'s parent switches from `main` to the backup `dev/alex_old_timestamp`. A day later, you want to refresh `dev/alex` with the latest data from `main`. You can't use Reset from Parent, since the backup is now the parent. Instead, use Branch Restore and select the original parent `main` as the source. 
```

Comment:
```suggestion
  For example, let's say you have a `main` branch with a child development branch `dev/alex`. You are working on `dev/alex` and decide to restore to an earlier point in time to fix something during development. At this point, `dev/alex`'s parent switches from `main` to the backup `dev/alex_old_timestamp`. A day later, you want to refresh `dev/alex` with the latest data from `main`. You can't use **Reset from parent**, since the backup is now the parent. Instead, use **Branch Restore** and select the original parent `main` as the source. 
```

Reply:
Seems that we should either link or bold here. Not entirely sure.